### ISSUE TITLE
[P2][Sourcing][Accessibility][Pretest][Table]The buttons "Search" and "Synchronize" are both marked with aria-hidden=true (i.e. they are hidden for the screen reader), but receive the focus with TAB, which leads to incorrect output.

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#10818]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
The buttons "Search" and "Synchronize" are both marked with aria-hidden=true (i.e. they are hidden for the screen reader), but receive the focus with TAB, which leads to incorrect output.

 aria-hidden should be removed

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)

Angular: 15
core/platform: 0.43.14

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
https://ng-15-downport--fundamental-ngx.netlify.app/#/platform/table/basic#multiple-row-selection

inspect sync and search icon buttons

**Comment by nikolay-kolarov** [comment]
Hello Colleagues,

The search button have defined shortcut for activation by the keyboard (it needs to be checked if F5 is implemented) and have to be excluded from the tab chain. Also the clear button must not be focusable, as you can use standard text input shortcuts to delete the input.
And aria-hidden has to be kept on them (exactly because they are not expected to be focused).

@droshev , @InnaAtanasova - FYI


Best Regards,
Nikolay
